src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2712,6 +2712,7 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_BeforeGetAndroidDependencies">
   <PropertyGroup>
     <_AndroidAllowMissingSdkTooling>True</_AndroidAllowMissingSdkTooling>
+    <AndroidGetJavaDependencies Condition=" '$(AndroidGetJavaDependencies)' == '' ">false</AndroidGetJavaDependencies>
```

Comment:
Looks good, but we also need to document the public property in the build-properties.md now :)

Reply:
Thanks, I've updated the property docs